certify-mock-identity.properties

```diff
@@ -13,21 +13,30 @@ mosip.certify.cache.store.individual-id=true
 mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
 mosip.certify.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
+mosip.certify.mock.vciplugin.issuer.key-cert=${mosip.certify.mock.vciplugin.issuer.cert}
```

Comment:
Can we avoid this property ?

Reply:
Removig the property as it is not referred anywhere